azure-pipelines.yml

```diff
@@ -251,6 +251,8 @@ stages:
 
     - job: runIntegrationTestsJob
       displayName: '.NET ITs'
+      workspace:
+        clean: all
```

Comment:
For .Net ITs we also need to clean up in order to refresh the expected and actual folder.